decidim-design/app/views/public/partials/_sidebar.html.erb

```diff
@@ -0,0 +1,71 @@
+<!-- Sidebar -->
+<div class="section">
+  <article class="card">
+    <div class="card__content card__content--center">
+      <button class="button button--sc success"><%= icon "check", class: "icon--small"%> Siguiendo</button>
+      <div class="">
+        Recibe una notificación cuando se publiquen nuevos artículos.
+      </div>
+    </div>
+  </article>
+</div>
+<div class="section">
+  <div class="section-heading">
+    Artículos con más actividad
+  </div>
+  <div class="m-bottom">
+    <strong>Lorem ipsum dolor sit amet, consectetur.</strong>
+    <div class="author-data author-data--nopadding">
```

Comment:
These classes are a bit confusing to me. How is author involved in this context? Maybe extending the `autor-data` class with the "no padding tweak" would be better?

Like

```scss
.comment-data {
  @extend .author-data;

  padding-left: 0
}
```

Reply:
Hehe, that's the reason I asked for feedback. I knew they were no related each other, but I wanted to reuse code. Then I used the BEM approach, but if you're more use to @extend norm, I'll keep in mind future times.
I'll update the code as you mentioned.

Reply:
Nono, using BEM notation is good, and it's consistent with naming of other parts of the current design, :+1: to that! Also :+1: to reusing code when it makes sense.

My concern is not about the style but about naming. If we're reusing this "author-data" block in other places completely unrelated to "authors", we should rename it to something else that better represent what this block is.

Reply:
I'm working on this, and a doubt comes up to me: where do I add it? 
Ok, I append it to `_author-avatar.scss`, easy peasy. But it's not right, that block is at the sidebar, but there's no kind of `sidebar.scss`, and it's not related to an author, as you said.
Or, should the small block (from `m-bottom` class) be a partial, perhaps?

Reply:
I had a second look to try to give you an answer and I'm noticing that the current naming already has this flaw. For example, the number of comments of a proposal are rendered under the `.author-data__extra` class, and they are not related to the proposal's author. So, I think this is something we should consider improving but I don't think you need to do that in this PR. Sorry for the noise :disappointed:.

Reply:
No worries, revert'n'go